staging/src/k8s.io/apiserver/pkg/server/egressselector/egress_selector.go

```diff
@@ -157,16 +157,23 @@ func (g *grpcProxier) proxy(ctx context.Context, addr string) (net.Conn, error)
 type proxyServerConnector interface {
```

Comment:
verified this diff results in identical content to release-1.24 branch for this file